db/migrate/20220216162803_create_articles.rb

```diff
@@ -0,0 +1,12 @@
+class CreateArticles < ActiveRecord::Migration[6.1]
+  def change
+    create_table :articles do |t|
+      t.text :title
+      t.text :description
```

Comment:
I think now we've called this "article" maybe "body" makes more sense for this field?